topic.go

```diff
@@ -319,6 +320,15 @@ func WithReadiness(ready RouterReady) PubOpt {
 	}
 }
 
+// WithLocalPublication option tells Publish to *only* notify local subscribers about a message.
+// This option prevents messages publication to peers.
```

Comment:
Lets improve the documentation a bit.

Reply:
> // WithLocalPublication returns a publishing option to notify in-process subscribers only.
> // It prevents message publication to mesh peers.
> // Useful in *edge* cases where the msg needs to be only delivered to the in-process subscribers, 
> // e.g. not to spam the network with outdated msgs.
> // Should not be used specifically for in-process pubsubing.

Tried to be consistent with the wording on other pub opts and incorporate your concern.

Reply:
Ok, will update the PR then

Reply:
Done